sklearn/decomposition/pca.py

```diff
@@ -388,26 +388,26 @@ def _fit(self, X):
         else:
             n_components = self.n_components
 
-        # Handle svd_solver
```

Comment:
Not a serious problem but I think you might leave the comment as it is. (``Handle _fit_svd_solver`` seems awkward from my side.)